analyzers/tests/SonarAnalyzer.UnitTest/Rules/OptionStrictOnTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿/*
```

Comment:
Scaffolding: Update of file `RSpecStrings.resx` is missing. Can you add those new lines from my original scaffolding branch?